components/alert/alert.css

```diff
@@ -0,0 +1,45 @@
+.alert {
+  display: flex;
+  align-items: center;
+  min-height: 2rem;
+  border-radius: 0.3rem;
+  padding: 0.75rem 1.25rem;
```

Comment:
use px for units against rem less than 1

Reply:
Done